<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prognosis</class>
 <widget class="QDialog" name="Prognosis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prognosis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="diagnosisLabel">
           <property name="text">
            <string>&amp;Diagnosis:</string>
           </property>
           <property name="buddy">
            <cstring>diagnosisComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="diagnosisComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="parameterValueTableWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="phaseTableWidget">
       <property name="columnCount">
        <number>2</number>
       </property>
       <column/>
       <column/>
      </widget>
      <widget class="QTableWidget" name="unknownParameterTableWidget">
       <property name="columnCount">
        <number>2</number>
       </property>
       <column/>
       <column/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
